cli/src/api/endpoints.rs

```diff
@@ -203,6 +208,14 @@ mod test {
         );
     }
 
+    #[test]
+    fn package_check_is_correct() {
+        assert_eq!(
+            package_check(API_URI).unwrap().as_str(),
+            format!("{API_URI}/{API_PATH}data/packages/check"),
+        );
+    }
```

Comment:
I absolutely hate these tests personally. I'm kinda curious how you feel about them. To me they're just pointless since there's no real danger of breakage here and it always requires making changes in two locations.

Reply:
I completely agree. I am happy to remove them (probably in a separate PR).